Auth/Constants/LoginOpCode.py

```diff
@@ -14,4 +14,6 @@ class LoginOpCode(Enum):
 
 class LoginResult(Enum):
     ''' Error codes '''
-    SUCCESS     = 0x00
+    SUCCESS             = 0x00
```

Comment:
Currently there are no handlers for all cases except success. I think it's not necessary to add this now.

Reply:
Ah gotcha.  Will wait to add them with the handlers then.